### PR TITLE
add wider_face dataset

### PR DESCRIPTION
**Pull request by lambdaflow**
Add WIDER FACE Dataset.

[Gist for dataset_info.json.](https://gist.github.com/lambdaflow/4f0a1e2110cf5dd7083fd7ea8786c603)

@us @Conchylicultor @cyfra

### CODE REVIEW

File: tensorflow_datasets/image/wider_face.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    ]
+
+  def _generate_examples(self, split, extracted_dirs):
+    """Yields examples."""
+    pattern_fname = re.compile(r'(.*.jpg)\n')
+    pattern_nbbox = re.compile(r'(\d+)\n')
+    pattern_annot = re.compile(r'(\d+) (\d+) (\d+) (\d+) (\d+) '
+                               r'(\d+) (\d+) (\d+) (\d+) (\d+) \n')
+    if split == 'test':
+      annot_file = 'wider_face_split/wider_face_test_filelist.txt'
```
Comment (us) [inline tensorflow_datasets/image/wider_face.py]: Don't use static path, please use `os.path.join`

Comment (lambdaflow) [reply tensorflow_datasets/image/wider_face.py]: > Don't use static path, please use `os.path.join`

switched to `os.path.join` in 8816908

**Comment by lambdaflow** [comment]
resolved setup.py conflict from master in 386c419

**Comment by breadbread1984** [comment]
please merge this patch. thx! I would like to use widerface with tfds.

**Pull request merged**